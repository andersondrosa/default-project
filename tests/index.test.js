import project from '../src'
it("Should be ok", function() {
  //
  expect(project).toMatchSnapshot();
});
