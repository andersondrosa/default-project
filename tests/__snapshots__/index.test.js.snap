// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be ok 1`] = `
Object {
  "test": "OK",
}
`;
